meson.build

```diff
@@ -47,4 +47,10 @@ pkgg.generate(
 
 fwupd_efi_dep = declare_dependency()
 
+# SBOM support
+uswid = find_program('uswid', required: false)
+if not uswid.found()
+  warning('uswid is unfound, so not embedding uSWID metadata')
+endif
```

Comment:
I think this needs to be a harder dependency now that there are reasons that you need to detect one version from another.

Reply:
Alternatively if uswid isn't around you can add some compatibility code that just puts a version string that can be picked up.  Then at some collective point we can drop that when python-uswid is readily available everywhere.

Reply:
> that just puts a version string

We did that in https://github.com/fwupd/fwupd-efi/commit/771a7c8e84641bfb4fc8673d8a55ab636f6a4994